lib/event-target.js

```diff
@@ -149,7 +153,7 @@ class MessageEvent extends Event {
    * @param {String} type The name of the event
    * @param {Object} [options] A dictionary object that allows for setting
    *     attributes via object members of the same name
-   * @param {*} [options.data=null] The message content
+   * @param {ArrayBuffer|Blob|Buffer|Buffer[]} [options.data=null] The message content
```

Comment:
It can actually be anything. When used in isolation

```js
const event = new MessageEvent('message', { data: NaN });
```

is perfectly valid. The same applies to `ErrorEvent`.

Reply:
Just to be sure, is the expectation/desire for this to be used in isolation?

Reply:
It is not, but it is a standalone module that can be used in isolation, so functions/features are documented as such.

Reply:
Got it. I will remove the commit, then.